.github/workflows/main.yml

```diff
@@ -383,13 +438,14 @@ jobs:
           fi
 
           npm publish *.tgz --tag=$tag --access=public --dry-run
-        env:
-          NPM_TOKEN: ${{ secrets.TOKEN }}
```

Comment:
Was this removal intentional?

Reply:
It wasn't, classic copy and paste gone wrong!